go/kbhttp/srv.go

```diff
@@ -103,6 +104,7 @@ func NewSrv(log logger.Logger, listenerSource ListenerSource) *Srv {
 	return &Srv{
 		log:            log,
 		listenerSource: listenerSource,
+		doneCh:         make(chan struct{}),
```

Comment:
should this be initialized in the `Start` method so the server could be started/stopped multiple times?

Reply:
Is that a usage pattern we've ever done? I can make it do that but it won't address the blocking on a nil channel.

Reply:
we usually have that flow in chat so it stuck out